templates/questions/index.html.ep

```diff
@@ -291,7 +291,10 @@ $(document).ready(function() {
 			Privacy Policy
 		</div>
 		<div class="privacyContent">
-			Here is the content of the privacy policy. It is kinda long, so we gotta make sure it wraps. Hopefully people really like this privacy policy. Hopefully it does not scare people away.
+			AMA uses "cookies" to enhance User experience. User's web browser places cookies on their hard drive for record-keeping purposes. Users are still completely anonymous. User may choose to set their web browser to refuse cookies, or to alert when cookies are being sent. If cookeis are disabled, note that some parts of the AMA site may not function properly.
```

Comment:
Explain "record-keeping purposes"?

Explain how users are anonymous when cookies are used?

cookeis should be spelled cookies.

If anonymity is a feature, why are cookies necessary?  Can you make the system behave the same without cookies?  What functionality is lost if cookies are disabled?